packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -41,53 +39,32 @@ export class PivotBase extends BaseComponent<IPivotProps, IPivotState> {
   private _pivotId: string;
   private focusZone = React.createRef<FocusZone>();
   private _classNames: { [key in keyof IPivotStyles]: string };
+  private _links: IPivotItemProps[] = [];
```

Comment:
Should `_links` trigger a rerender? Should it not have been in state originally?

Reply:
`_links` shouldn't be in state because it's just a cached calculation. Ideally it should be memoized--which is essentially what JD is doing here, but I'm not sure I like the way it's currently implemented.